apps/web/src/modules/proposal/components/ProposalVotes/VoterParticipation.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Text } from '@zoralabs/zord'
+import { useMemo } from 'react'
+
+import { useLayoutStore } from 'src/stores'
+
+import { VoterParticipationVariants } from './VoterParticipation.css'
+
+export interface VoterParticipationProps {
+  totalVotes: number
+  maxVotes: number
+}
+
+export const VoterParticipation: React.FC<VoterParticipationProps> = ({
+  totalVotes,
+  maxVotes,
+}) => {
+  const { isMobile } = useLayoutStore()
+  const participation = (totalVotes / maxVotes) * 100
+
+  const variant = useMemo(() => {
+    if (participation >= 65) return 'positive'
```

Comment:
nit: what do these values represent? maybe worth to assign them as a denoted const?